src/transformers/feature_extraction_utils.py

```diff
@@ -121,6 +121,9 @@ def convert_to_tensors(self, tensor_type: Optional[Union[str, TensorType]] = Non
 
         # Convert to TensorType
         if not isinstance(tensor_type, TensorType):
+            if tensor_type == "flax":
```

Comment:
Do we need that? I'm a bit worried that this aliasing doesn't exist in other spots and that it'll trigger some bugs from users

Reply:
We don't **need** that, but not having it forces in ALL pipelines to have

`return_framework="jax" if self.framework == "flax" else self.framework`
which is very tedious. 
As a user, I found it very confusing that all models are named `FlaxXXX` and this expected `jax`. 
Ofc I figured out how to solve, but seems unlikely (to me) that I would use `jax` tensors with `pythorch` models or anything like that (or use other tensor types within flax, even if numpy works)

Reply:
I understand what you're saying, but this is basically a way to ask for aliasing `flax` where possible (and where surprising)